tests/unit/test_tts_stream_normalizer.py

```
@@ -0,0 +1,266 @@
+    上一轮的尾部空格不应被带到下一轮的首个 chunk 里。
+    """
+    n = TtsStreamNormalizer()
+    n.feed("hello ")  # 上一轮 trailing 暂存
+    # 上一轮的尾部空格在 flush 时丢掉
+    n.flush()
+    # 新轮次模拟 reset
+    n.reset()
+    assert n.feed("world") == "world"
+
+
+# --- 端到端：拼接全部 emit == 在"完整文本"上跑 replace_blank ------------
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)